src/hooks/core/chat/useChatUtilsHook.ts

```diff
@@ -205,6 +266,7 @@ function useChatUtilsHook(): UseChatUtilsHook {
   };
 
   return {
+    isLoadingFetchingChannelDetail,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Exposing `isLoadingFetchingChannelDetail` in the return statement allows other components or hooks to access this state and react accordingly, such as showing a loading spinner while fetching channel details. This is a good practice as it improves user experience by providing feedback about the loading status.
```diff
  return {
+    isLoadingFetchingChannelDetail,
      selectedChannel,
      selectedChannelKey,
      goBack,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->